docs/guide/tutorials.rst

```diff
@@ -0,0 +1,244 @@
+	This file contains component group names and their properties such as ``retention_time`` and ``intensity`` with the
+	header in the following format :
+	``component_group_name,n_heavy_l ,n_heavy_u,n_light_l,n_light_u,n_detecting_l,``
+	``n_detecting_u,n_quantifying_l,n_quantifying_u,n_identifying_l,n_identifying_u,``
+	``n_transitions_l,n_transitions_u,ion_ratio_pair_name_1,ion_ratio_pair_name_2,``
+	``ion_ratio_l,ion_ratio_u,retention_time_l,retention_time_u,intensity_l,``
+	``intensity_u,overall_quality_l,overall_quality_u,retention_time_l_,``
+	``retention_time_u_``.
+	A sample row may look like the following :
+	``Serotonin,0,10,0,10,0,10,0,10,0,10,0,10,,,,,1.37,2.37,``
+	``-1.00E+12,1.00E+15,-1.00E+12,1.00E+12,82.2,142.2``.
+
+#. featureFilterComponents.csv
+	This file contains component names and their properties such as 
+	``retention_time`` and ``intensity``. 
+	The header is defined as follows :
+	``component_name,intensity_l,intensity_u,overall_quality_l,overall_quality_u,``
+	``retention_time_l,retention_time_u``.
+	A sample row may be written as follows :
+	``Serotonin,100,1.00E+15,-500,1.00E+12,0.87,2.87``
+
+#. featureQCComponentGroups.csv
+	This file contains QC component group names and their properties such as 
+	``retention_time`` and ``intensity``.
+	The header is defined as follows :
+	``component_group_name,n_heavy_l ,n_heavy_u,n_light_l,n_light_u,n_detecting_l,``
+	``n_detecting_u,n_quantifying_l,n_quantifying_u,n_identifying_l,n_identifying_u,``
+	``n_transitions_l,n_transitions_u,ion_ratio_pair_name_1,ion_ratio_pair_name_2,``
+	``ion_ratio_l,ion_ratio_u,retention_time_l,retention_time_u,``
+	``intensity_l,intensity_u,overall_quality_l,overall_quality_u``.
+
+	A sample row may be written as follows :
+	``Serotonin,0,10,0,10,1,10,1,10,0,10,1,``
+	``10,,,,,1.37,2.37,0,1.00E+15,-500,1.00E+12``.
+
+#. featureQCComponents.csv
+	* Purpose : This file contains QC component  names and their properties such as ``retention_time`` and ``intensity``. 
+	The header is defined as follows :
+	``component_name,metaValue_peak_apex_int_l,metaValue_peak_apex_int_u,``
+	``metaValue_logSN_l,metaValue_logSN_u,metaValue_total_width_l,``
+	``metaValue_total_width_u,metaValue_width_at_50_l,metaValue_width_at_50_u,``
+	``metaValue_tailing_factor_l,metaValue_tailing_factor_u,``
+	``metaValue_asymmetry_factor_l,metaValue_asymmetry_factor_u,``
+	``metaValue_baseline_delta_2_height_l,metaValue_baseline_delta_2_height_u,``
+	``metaValue_points_across_baseline_l,metaValue_points_across_baseline_u,``
+	``metaValue_points_across_half_height_l,metaValue_points_across_half_height_u,``
+	``metaValue_logSN_l,``
+	``metaValue_logSN_u,intensity_l,intensity_u,overall_quality_l,overall_quality_u``.
+
+	A sample row may be written as follows : 
+	``Serotonin,0,5.00E+06,1,1.00E+06,0.1,1,0.001,0.25,0,2,0.8,2.5,``
+	``-0.25,0.25,20,500,5,500,1,1.00E+06,0,1.00E+15,0,1.00E+12``.
+
+#. features
+	This folder contains features info in the 
```

Comment:
Can you add a little bit more hear about what the feature files are?  e.g., "Features contain the processed results of a single sample after applying a workflow and running the command `STORE_FEATURES`".  Also it would be important to note that the Features can be stored at both the sample (i.e., injection) or the sample group (merged injections) level.